tests/transfers.rs

```diff
@@ -767,3 +771,173 @@ fn tapret_wlt_receiving_opret() {
         1000,
     );
 }
+
+fn print_history(history: &HashMap<XWitnessId, IfaceOp<AmountChange>>) {
```

Comment:
please move this into `tests/utils/helpers.rs` since this file should contain only tests. also please rename this method into `debug_fungible_history`